pkg/apis/v1alpha1/serviceexport.go

```diff
@@ -88,3 +92,159 @@ type ServiceExportList struct {
 	// +listType=set
 	Items []ServiceExport `json:"items"`
 }
+
+// ServiceExportConditionType is a type of condition associated with a
+// ServiceExport. This type should be used with the ServiceExportStatus.Conditions
+// field.
+type ServiceExportConditionType string
+
+// ServiceExportConditionReason defines the set of reasons that explain why a
+// particular ServiceExport condition type has been raised.
+type ServiceExportConditionReason string
+
+// NewServiceExportCondition creates a new ServiceExport condition
+func NewServiceExportCondition(t ServiceExportConditionType, status metav1.ConditionStatus, reason ServiceExportConditionReason, msg string) metav1.Condition {
+	return metav1.Condition{
+		Type:               string(t),
+		Status:             status,
+		Reason:             string(reason),
+		Message:            msg,
+		LastTransitionTime: metav1.Now(),
+	}
+}
+
+const (
+	// ServiceExportConditionAccepted is true when the Service Export is accepted.
+	// This does not indicate whether or not the configuration has been exported
+	// to a control plane / data plane.
+	//
+	//
+	// Possible reasons for this condition to be true are:
+	//
+	// * "Accepted"
+	//
+	// Possible reasons for this condition to be False are:
+	//
+	// * "NoService"
+	// * "InvalidServiceType"
+	//
+	// Controllers may raise this condition with other reasons,
+	// but should prefer to use the reasons listed above to improve
+	// interoperability.
+	ServiceExportConditionAccepted ServiceExportConditionType = "Accepted"
+
+	// ServiceExportReasonAccepted is used with the "Accepted" condition when the
+	// condition is True.
+	ServiceExportReasonAccepted ServiceExportConditionReason = "Accepted"
+
+	// ServiceExportReasonNoService is used with the "Accepted" condition when
+	// the associated Service does not exist.
+	ServiceExportReasonNoService ServiceExportConditionReason = "NoService"
```

Comment:
Is there potentially any RBAC reason that (controller can't read Service resources for some reason) that could cause this?
```suggestion
	// ServiceExportReasonNoService is used with the "Accepted" condition when
	// the Service to be exported could not be found.
	ServiceExportReasonServiceNotFound ServiceExportConditionReason = "ServiceNotFound"
```